content/introduction.html

```diff
@@ -17,20 +17,24 @@ <h1>About the program</h1>
   Waterloo in any way.
 </p>
 
-<!-- // todo: robbie add some class photos  -->
 <div class="vb"></div>
 <div class="class-photo">
   <img src="./img/class.png" />
+  <div class="vbx2"></div>
+  <p class="full">
+    For more photos of the class of 2022, check out the yearbook!
+    <a target="_blank" href="https://sexxii.github.io/yearbook/">
+      https://sexxii.github.io/yearbook/</a
+    >
+  </p>
 </div>
-<div class="vbx6"></div>
+<div class="vbx2"></div>
 
 <h1>Preface</h1>
 
 <p class="full">
   The findings in this report were derived from the results of a class survey.
-  In total, 66 out of 92 (72%) graduating students, and 7 who were in SE but
-  transferred out, responded.
-  <!-- // todo: robbie check these numbers  -->
+  In total, 66 graduating students responded.
```

Comment:
Do we not know how many students graduated/moved out?

Reply:
i have no clue. but just asked se advisor today, let's see what they say!

Reply:
101!